extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -649,20 +654,23 @@ export class AmpA4A extends AMP.BaseElement {
         // This block returns the ad creative and signature, if available; null
```

Comment:
Let's update these comments to reflect that size will also be passed down through here.